yjit/src/stats.rs

```diff
@@ -404,10 +404,12 @@ fn rb_yjit_gen_stats_dict() -> VALUE {
     }
 
     // If the stats feature is enabled
-    #[cfg(feature = "stats")]
+
     unsafe {
         // Indicate that the complete set of stats is available
         rb_hash_aset(hash, rust_str_to_sym("all_stats"), Qtrue);
+        #[cfg(feature = "stats")]
+        rb_hash_aset(hash, rust_str_to_sym("vm_insns_count_available"), Qtrue);
```

Comment:
Minor detail but, can't you directly check whether `:vm_insns_count` exists as a key in the hash rather than rely on a separate boolean value?

Reply:
Nope, the key is in there whether we count or not. It is just set to 0. Which also feels wrong to check. I mean maybe we will always have an instruction run there? But seems a bit too implicit to me.

Reply:
We could not set the key when not in a stats build?

Reply:
I now skip adding the key if it is not a stats build.